src/lib/libfs.js

```diff
@@ -1843,7 +1846,15 @@ FS.staticInit();`;
       // Add a function that defers querying the file size until it is asked the first time.
       Object.defineProperties(node, {
         usedBytes: {
-          get: function() { return this.contents.length; }
+          get: function() { return this.contents.length; },
+          set: function(newLength) {
+            // Resize the buffer by allocating new.
+            if (newLength != this.contents.length) {
+              var newBuffer = new Uint8Array(newLength);
+              newBuffer.set(this.contents);
+              this.contents = newBuffer;
+            }
```

Comment:
This seems like it does more than just fix the "cannot assign to `usedBytes` in strict mode" though.    it also changes what the assignment does, even in non-strict mode. 

In non-strict mode what does the assignment do?  Is it a no-op?